test/e2e/framework/util.go

```diff
@@ -538,6 +538,23 @@ func logPodStates(pods []v1.Pod) {
 	Logf("") // Final empty line helps for readability.
 }
 
+// logPodTerminationMessages logs termination messages for failing pods.  It's a short snippet (much smaller than full logs), but it often shows
```

Comment:
Looks like you pulled in some changes from another branch by accident?

Reply:
it was picked up in a rebase.

Reply:
It should be removed...

Reply:
^ agreed, rebase should not have pulled in this diff 🤔